packages/pancake-uikit/src/components/DropdownSearch/DropdownSearch.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useRef, useEffect } from "react";
+import styled, { css } from "styled-components";
+import { Input } from "../Input";
+import { Text } from "../Text";
+import { ArrowDropDownIcon } from "../Svg";
+import { DropdownSearchOptionProps, DropdownSearchProps } from "./types";
+
+const StyledInput = styled(Input)`
+  border-radius: 16px;
```

Comment:
Input already has a border radius of 16px

Reply:
Removed